packages/fxa-content-server/app/scripts/views/settings/subscription.js

```diff
@@ -25,27 +26,8 @@ const View = FormView.extend({
     }
   },
 
-  template() {
-    return '';
-  },
-
   afterRender() {
-    return this._hasActiveSubscriptions().then(hasSubs => {
-      if (hasSubs) {
-        this.$el.html(this.renderTemplate(Template));
-      }
-    });
-  },
-
-  /**
-   * Check to see if the account has any active subscriptions.
-   *
-   * @returns {Promise} resolves to a bool.
-   */
-  _hasActiveSubscriptions() {
-    return this.getSignedInAccount()
-      .fetchActiveSubscriptions()
-      .then(subscriptions => subscriptions.length > 0);
+    return this.getSignedInAccount().hasSubscriptions();
```

Comment:
The rendering related changes here is from @shane-tomlinson [pointing out that we can control view rendering in `afterRender`](https://github.com/mozilla/fxa/pull/2079#discussion_r311106508).